DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/svd/src/golden_pca.hpp

```diff
@@ -0,0 +1,316 @@
+#include <math.h>
+
+#include <fstream>
+#include <iomanip>
+#include <random>
+
+/*
+This file implements the steps to
+identify principal components (Eigen vectors)
```

Comment:
~eigen vector~ eigenvector
~eigen value~ eigenvalue

please fix this in other source files